### PR TITLE
Add Paul Kerensa

### PR DESCRIPTION
**Pull request by BenWoodford76**
Created a person for Paul Kerensa

Solves Issue 665

### CODE REVIEW

File: _people/paul_kerensa.mdown [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  - Comedy Writer
+  - Actor
+links:
+  - type: Personal Website
+    href: "http://www.paulkerensa.com/"
+  - type: Twitter
+    username: paulkerensa
+---
+
+Since Graduating, Paul has gone on to have an extensive career in both Television and Radio. His Writing credits include the BBC sitcoms Not Going out and Miranda, and he is a regular contributor to BBC Radio 2's Pause for Thought. Paul has won numerous awards for both his writing and performing, including ITV's Take The Mike stand-up award, A British Comedy Award for Miranda and a BAFTA also for Miranda. Paul is a regular up at the Edinburgh Fringe, and performs at both comedy clubs and churches.
```
Comment (wjdp) [inline _people/paul_kerensa.mdown]: This is going to be very pernickety. You should configure your text editor to automatically put a blank empty line at the end of the file. It's what that symbol is showing (no newline at EOF).

https://robots.thoughtbot.com/no-newline-at-end-of-file

Comment (nhyty) [inline _people/paul_kerensa.mdown]: Where is this setting?

Comment (wjdp) [inline _people/paul_kerensa.mdown]: See the link, it explains for Sublime. Other editors vary.

Comment (wjdp) [inline _people/paul_kerensa.mdown]: Oh, poo. Probably won't break much but more importantly @BenWoodford76: we use `.md` for markdown instead of `.mdown`.

**Pull request merged**